src/elf/dyn.rs

```diff
@@ -10,6 +10,9 @@ macro_rules! elf_dyn {
             /// Integer value
             pub d_val: $size,
         }
+
+        use plain;
+        unsafe impl plain::Plain for Dyn {}
```

Comment:
Another problem is that I need to inspect the definition of referenced types as well, otherwise the derive would only work for the most trivial cases. I think I'd have to use unstable rustc interfaces to do that.